mypy.ini

```diff
@@ -15,7 +15,5 @@ warn_redundant_casts = True
 
 [mypy-hypothesis.internal.*]
 ignore_errors = True
-
-# Annotating PY2-only code not worth the trouble for just a few months
```

Comment:
I note that this comment was removed by not the thing it was commenting. What's up with that?

Reply:
`hypothesis.vendor` is excluded from e.g. our formatting job; if you'd prefer that I just make `mypy` pass on the pretty-printing code that shouldn't be too hard.